.env

```
@@ -1,7 +1,7 @@
-DB_HOST="ec2-52-3-60-53.compute-1.amazonaws.com"
+DB_HOST="otto.db.elephantsql.com"
 DB_PORT=5432
-DB_USER="xybvtiinusasyh"
-DB_PASSWORD="389e9dac39c30290ed036e62362ec3e7cc435f0432f3b2cdce1bf6125bb61ae3"
-DB_NAME="d63vklqd241lbk"
+DB_USER="rutelazj"
+DB_PASSWORD="ZN1uXyCCGjXbIs5KR-3NlLSJjVrPFVTT"
+DB_NAME="rutelazj"
```

```suggestion
DB_USER="xybvtiinusasyh"
DB_PASSWORD="389e9dac39c30290ed036e62362ec3e7cc435f0432f3b2cdce1bf6125bb61ae3"
DB_NAME="d63vklqd241lbk"
```

Lets revert this until dicember